.circleci/config.yml

```diff
@@ -0,0 +1,149 @@
+version: 2
```

Comment:
Does this new file remove the need for `run-tests.sh`?

Reply:
yes